dae/dae/configuration/gpf_config_parser.py

```diff
@@ -117,7 +117,8 @@ def parse_and_interpolate(
             content: str, parser=yaml.safe_load,
             conf_dir=None) -> dict:
         """Parse text content and perform variable interpolation on result."""
-        interpol_vars = parser(content).get("vars", {})
+        parsed_content = parser(content) or {}
```

Comment:
Why are we doing this now?

Reply:
If the content is an empty string (""), the `yaml.safe_load` would return `None`, breaking the downstream code.